pybamm/expression_tree/averages.py

```diff
@@ -38,9 +39,22 @@ def __init__(self, child: pybamm.Symbol) -> None:
         integration_variable = x
         super().__init__(child, "x-average", integration_variable)
 
-    def _unary_new_copy(self, child: pybamm.Symbol):
-        """See :meth:`UnaryOperator._unary_new_copy()`."""
-        return x_average(child)
+    def _unary_new_copy(
+        self, child: pybamm.Symbol, perform_simplifications: bool = True
+    ):
+        """
+        Creates a new copy of the operator with the child `child`.
+
+        Uses the convenience function :meth:`x_average` to perform checks before
+        creating an XAverage object.
+        """
+        if perform_simplifications:
+            return x_average(child)
+        else:
+            raise NotImplementedError(
+                f"{self.__class__.__name__} should always be copied using "
```

Comment:
why do we have to use simplification checks here?

Reply:
and others below....

Reply:
Turns out we didn't, I've removed these to follow the same format as other overwritten functions